src/queues/job-queue.js

```diff
@@ -85,7 +85,7 @@ const initQueue = () => {
   });
 
   console.info('deleteActivityProcessQueue job is working!');
-  deleteActivityProcessQueue.process(deleteActivityProcessJob);
+  deleteActivityProcessQueue.process(removeActivityProcessJob);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `deleteActivityProcessQueue.process()` now calls `removeActivityProcessJob` instead of `deleteActivityProcessJob`. Ensure that `removeActivityProcessJob` is correctly defined and imported, otherwise this will result in a reference error.
```diff
- deleteActivityProcessQueue.process(deleteActivityProcessJob);
+ deleteActivityProcessQueue.process(removeActivityProcessJob);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->